<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeKey</class>
 <widget class="QDialog" name="ChangeKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
color:rgb(0, 255, 255);
background:rgb(0, 255, 255);
font-size:15px;
font-style:MingLiU-ExtB;
}
QWidget{
border:2px sold rgb(0, 255, 255);
background:rgb(0, 0, 40);
}
QLineEdit{
color:rgb(0, 255, 255);
background-color:#405361;
height:40px;
border-style:outset;
border-radius:10px;
border:2px solid rgb(0, 255, 255);
font-size:16px;
font-style:MingLiU-ExtB;
}
QPushButton{
width:150px;
height:40px;
border-style:outset;
border-radius:10px;
border:2px solid rgb(0, 255, 255);
font-size:16px;
font-style:MingLiU-ExtB;
}
QPushButton:pressed{
color:rgb(0, 0, 50);
background-color:rgb(0, 255, 255);
border-style:inset;
font-style:MingLiU-ExtB;
}
QLabel{
color:rgb(0, 255, 255);
background:rgba(85,170,255,0);
height:40px;
font-size:16px;
font-style:MingLiU-ExtB;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <property name="spacing">
    <number>10</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="ChangeKeyBtnLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ChangeKeyAckBtn">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ChangeKeyCancerBtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QFormLayout" name="ChangeKeyInputLayout">
     <property name="labelAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="horizontalSpacing">
      <number>10</number>
     </property>
     <property name="verticalSpacing">
      <number>10</number>
     </property>
     <property name="leftMargin">
      <number>10</number>
     </property>
     <property name="topMargin">
      <number>20</number>
     </property>
     <property name="rightMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item row="0" column="1">
      <widget class="QLineEdit" name="UsernameChangeEdit">
       <property name="maxLength">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="UserKeyBeforeLabel">
       <property name="text">
        <string>原密码：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="UserKeyBeforeEdit">
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="UserKeyAfterLabel">
       <property name="text">
        <string>新密码：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="UserKeyAfterEdit">
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="UserKeyAckLabel">
       <property name="text">
        <string>确认密码：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="UserKeyAckEdit">
       <property name="maxLength">
        <number>10</number>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="UsernameChangeLabel">
       <property name="text">
        <string>用户名：</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
